modules/ossm-auto-route.adoc

```diff
@@ -26,7 +26,7 @@ spec:
     - bookinfo.example.com
 ----
 
-Then, the following OpenShift Routes are created automatically. You can check that the routes are created with the following command. In this example, `istio-system` is the name of the control plane project.
+Then, the following OpenShift Routes are created automatically. You can check that the routes are created with the following command. In this example, `istio-system` is the name of the {SMProductShortName} control plane project.
```

Comment:
```suggestion
This `Gateway` resource creates the following OpenShift Routes. You can check that the routes are created by using the following command. In this example, `istio-system` is the name of the {SMProductShortName} control plane project.
```
Making this less of a step so that this module doesn't cross over into being a procedure module.

Reply:
done